Pat.Sender/MessageSender.cs

```diff
@@ -42,7 +43,15 @@ public async Task SendMessages(IEnumerable<Message> messages)
                 var connectionString = _connectionResolver.GetConnection();
                 try
                 {
-                    var client = TopicClientResolver.GetTopic(connectionString, _senderSettings.EffectiveTopicName);
+                    TopicClient client ;
+                    if (_senderSettings.TokenProvider != null) {
```

Comment:
Sorry, pedantic one. Can we keep the opening brace on a new line to keep consistency please :)